Gemfile

```diff
@@ -41,10 +39,17 @@ group :development, :test do
   gem 'rspec-rails', '~> 5.0.0'
 end
 
+gem 'sqlite3', :group => [:development, :test]
+group :production do
+  gem 'thin'
+  gem 'pg'
+end
+
 group :development do
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'pq'
```

Comment:
You add `sqlite3` for [:development, :test] and then `pg` for :development, why?

Reply:
Some gem required sqlite3 to be installed.